jobs/pingcap/tidb/latest/merged_integration_br_test.groovy

```diff
@@ -1,5 +1,6 @@
 // REF: https://<your-jenkins-server>/plugin/job-dsl/api-viewer/index.html
 pipelineJob('pingcap/tidb/merged_integration_br_test') {
+    disabled(true)
```

Comment:
How about controlling by prow bot to:
- report or not report git commit status
- run on not run

Reply:
pipeline test found unstable, need to debug, prow is not available for takeover at this time